init-git.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-TFM_GIT_SHA=TF-Mv1.1
-TFM_TESTS_GIT_SHA=TF-Mv1.1
+TFM_GIT_SHA=8bebd05745a8b27dccc6403f0215fa6e39de3bc1
+TFM_TESTS_GIT_SHA=652bb8a1502c95bfd1ee8e44ccd1126ce74ae41f
 MCUBOOT_GIT_SHA=v1.6.0
 MBED_CRYPTO_GIT_SHA=mbedcrypto-3.0.1
```

Comment:
IIRC, mbedcrypto is no longer used. Instead they use mbedtls 2.23.0 I think.

Update README.rst as well.

Reply:
Yeah, I've thought about this. The thing is, I am not currently removing the MBedCrypto directory, that's why I kept the entry here, and in the readme. 
On second thought, I'll start with removing the info from the init-git and readme, and then we can think of cleaning up the directories on the next upmerge.

Reply:
Applied.